samples/drivers/video/capture/src/main.c

```diff
@@ -125,10 +120,30 @@ int main(void)
 		return 0;
 	}
 
-	printk("- Default format: %c%c%c%c %ux%u\n", (char)fmt.pixelformat,
+	if (CONFIG_VIDEO_FORMAT_HEIGHT) {
+		fmt.height = CONFIG_VIDEO_FORMAT_HEIGHT;
+	}
+
+	if (CONFIG_VIDEO_FORMAT_WIDTH) {
+		fmt.width = CONFIG_VIDEO_FORMAT_WIDTH;
```

Comment:
wouldn't it be better (less expensive) to use `#if CONFIG_VIDEO_FORMAT_WIDTH` instead of `if (CONFIG_VIDEO_FORMAT_WIDTH)`?